examples/apps/ai-collab/src/components/UserPresenceGroup.tsx

```diff
@@ -69,23 +58,35 @@ const UserPresenceGroup: React.FC<UserPresenceProps> = ({
 				// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
 				const photoBlob = await client.api("/me/photo/$value").get();
 				// eslint-disable-next-line @typescript-eslint/no-unsafe-argument
-				const photoUrl = URL.createObjectURL(photoBlob);
-				setPhotoUrls((prevPhotos) => {
-					return [...prevPhotos, photoUrl];
-				});
-				userPresenceGroup.onlineUsers.local.set(`id-${uuid()}`, {
-					value: { photo: photoUrl },
-				});
-				setIsPhotoFetched(true);
+				photoUrl = URL.createObjectURL(photoBlob);
```

Comment:
Nit: the logic for the photo in SPE case I think wants to move to src/infra/graphHelper.ts, which basically encapsulates the calls to the graph API. We can do it as a follow up PR.

Reply:
Moved to helper files.